hack/make/test-docker-py

```diff
@@ -12,7 +12,7 @@ set -e
 	}
 
 	# exporting PYTHONPATH to import "docker" from our local docker-py
-	test_env PYTHONPATH="$dockerPy" python "$dockerPy/tests/integration_test.py"
+	test_env PYTHONPATH="$dockerPy" NOT_ON_HOST=true python "$dockerPy/tests/integration_test.py"
```

Comment:
A minority of integration tests don't run properly when executed inside a container, so they're skipped when NOT_ON_HOST is set to true.

Ctrl-F NOT_ON_HOST https://github.com/docker/docker-py/blob/master/tests/integration_test.py for more details :)